salt/modules/zabbix.py

```diff
@@ -232,7 +232,7 @@ def _connarg(name, key=None):
                     name = name[len(prefix) :]
                 except IndexError:
                     return
-            val = __salt__["config.option"]("zabbix.{}".format(name), None)
+            val = __salt__["config.get"]("zabbix:{}".format(name), None)
```

Comment:
This would break documented behavior. The docs state the user can set the values:

```
zabbix.user: Admin
zabbix.password: mypassword
zabbix.url: http://127.0.0.1/zabbix/api_jsonrpc.php
```

In either the minion config or pillar. If you change the delimiter to `:` this means the minion configuration would no longer work.

Reply:
Hmmm so I didn't understand how should work and misguided the #59338  original poster. Should have waited the bug to be triaged. As the #59338 I did understand that `user`, `password` and `url` were nested inside `zabbix`.

Reply:
We could still support the nested pillar as well. Looks like the `slack_notify.py` module is doing this here: https://github.com/saltstack/salt/blob/v3002.6/salt/modules/slack_notify.py#L48

You could pass in `or __salt__["config.get"]("zabbiz:{}")".format(name), None)` and that would allow it to support both the old configuration and to use nested pillar data.

Reply:
Nice, I'll see that at night and fix the fix to allow both syntaxes

Reply:
Ok, did the change and now both syntaxes are allowed, updated the unit tests accordingly.

The test jobs are failing for `pr-centos7-py3-tcp-pytest` and MacOSX, but at first sight I can't find a relationship between this last change and the failures.